framework/src/main/java/org/checkerframework/framework/type/SubtypeVisitHistory.java

```diff
@@ -8,8 +8,7 @@
 import org.checkerframework.javacutil.Pair;
 
 /**
- * THIS CLASS IS DESIGNED FOR USE WITH DefaultTypeHierarchy, DefaultRawnessComparer, and
- * StructuralEqualityComparer ONLY.
+ * THIS CLASS IS DESIGNED FOR USE WITH DefaultTypeHierarchy and StructuralEqualityComparer ONLY.
```

Comment:
This should be reverted.   (Rawness here is referring to a generic type without a type argument.)